### DISCUSSION TITLE
Cannot publish unborn Head with Unreal Engine source control

### DISCUSSION THREAD

**Discussion by muratcali** [community/community#21706] [New to GitHub]
<p>Hello everyone,</p>
<p>Im working on a Unreal Engine project with my team and Im getting some issues with the github desktop app. It says that i cannot publish my repository because of a unborn Head… I have search on Google to find some solutions to fix this problem, but there was no success. I also tried to add a summary, and i also deleted the automatically generated index.lock file by Unreal Engine himself. But no succes… Can someone help me with this problem? </p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/9d89eacabe2285529e873f81d182ffe90e5dbc99.png" data-download-href="/uploads/short-url/mtErPYofCrBdrmlnU12jLGX8OTf.png?dl=1" title="Screenshot_8.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/38989039/181066411-d0b15df9-d236-4d37-9ca5-7aa45d91671c.png" alt="Screenshot_8.png" data-base62-sha1="mtErPYofCrBdrmlnU12jLGX8OTf" width="690" height="472" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/9d89eacabe2285529e873f81d182ffe90e5dbc99_2_690x472.png, https://aws1.discourse-cdn.com/github/original/1X/9d89eacabe2285529e873f81d182ffe90e5dbc99.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/9d89eacabe2285529e873f81d182ffe90e5dbc99.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/9d89eacabe2285529e873f81d182ffe90e5dbc99_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot_8.png</span><span class="informations">961×658 36.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by steveward** [comment, marked as answer]
<p>Hi <a class="mention" href="/unactive">@unactive</a> – I replied to your question via email when you wrote in. If you are still encountering any issues, please let me know and we can continue to investigate this!</p>
